index.html

```diff
@@ -11,15 +11,16 @@
 
 <body>
   <header>
-    <h1>Awesome Books</h1>
+    <h1> All Awesome Books</h1>
   </header>  
    
 
   <section class="Books">
+    
   </section>
 
-
   <section class="AddingBook">
+    <h1>Add a New Book</h1>
     <form class="form">
```

Comment:
Please follow the wireframe for the form to add books.  The button should be on the right side of the form and please give a bit more height and width to input fields.